windows/rngallery/MainPage.cpp

```diff
@@ -16,5 +16,6 @@ namespace winrt::rngallery::implementation
         InitializeComponent();
         auto app = Application::Current().as<App>();
         ReactRootView().ReactNativeHost(app->Host());
+        ReactRootView().IsPerspectiveEnabled(false);
```

Comment:
Include a comment with a link to the RNW issue #?

Reply:
Will do!

Reply:
> Will do!

In my react-native-windows project, I add ReactRootView().IsPerspectiveEnabled(false); in two files below:
1. myproject/windows/myproject/MainPage.cpp
2. myproject/node_modules/react-native-windows/template/cpp-app/src/MainPage.cpp
but it still can't scroll inside the webview.